recipes-libretro/dolphin-libretro/dolphin-libretro_git.bb

```diff
@@ -10,41 +10,40 @@ require files/dolphin-32bit-configuration.inc
 
 LIBRETRO_GIT_REPO = "github.com/libretro/dolphin.git"
 
+SRC_URI:append = " file://0001-JitArm64-Fix-improper-uses-of-offsetof.patch"
+
 DEPENDS = " \
   ${@bb.utils.contains('DISTRO_FEATURES', 'retroarch-opengl', 'virtual/libgl ', '', d)} \
-  spirv-tools \
+  curl \
+  fmt \
   libusb \
+  pugixml \
+  xz \
+  zlib \
 "
 
 PACKAGECONFIG ?=  " \
   ${@bb.utils.contains('DISTRO_FEATURES', 'retroarch-gles', 'egl', '', d)} \
   ${@bb.utils.contains('DISTRO_FEATURES', 'retroarch-gles3', 'egl', '', d)} \
-  ${@bb.utils.filter('DISTRO_FEATURES', 'llvm vulkan x11', d)} \
+  ${@bb.utils.filter('DISTRO_FEATURES', 'vulkan x11', d)} \
   evdev \
-  sdl \
-  libretro \
+  sdl2 \
 "
 
-PACKAGECONFIG[alsa] = "-DENABLE_ALSA=ON,-DENABLE_ALSA=OFF"
-PACKAGECONFIG[analytics] = "-DENABLE_ANALYTICS=ON,-DENABLE_ANALYTICS=OFF"
-PACKAGECONFIG[discord] = "-DUSE_DISCORD_PRESENCE=ON,-DUSE_DISCORD_PRESENCE=OFF"
 PACKAGECONFIG[dsptool] = "-DDSPTOOL=ON,-DDSPTOOL=OFF"
 PACKAGECONFIG[egl] = "-DENABLE_EGL=ON,-DENABLE_EGL=OFF,virtual/egl virtual/libgl virtual/libgles2"
-PACKAGECONFIG[evdev] = "-DENABLE_EVDEV=ON,-DENABLE_EVDEV=OFF"
+PACKAGECONFIG[evdev] = "-DENABLE_EVDEV=ON,-DENABLE_EVDEV=OFF,libevdev"
 PACKAGECONFIG[fastlog] = "-DENCODE_FRAMEDUMPS=ON,-DENCODE_FRAMEDUMPS=OFF"
-PACKAGECONFIG[framedumps] = "-DENCODE_FRAMEDUMPS=ON,-DENCODE_FRAMEDUMPS=OFF"
 PACKAGECONFIG[gdbstub] = "-DGDBSTUB=ON,-DGDBSTUB=OFF"
 PACKAGECONFIG[gprof] = "-DENCODE_FRAMEDUMPS=ON,-DENCODE_FRAMEDUMPS=OFF"
 PACKAGECONFIG[headless] = "-DENABLE_HEADLESS=ON,-DENABLE_HEADLESS=OFF"
-PACKAGECONFIG[libretro] = "-DLIBRETRO=ON -DLIBRETRO_STATIC=1,-DLIBRETRO=OFF"
```

Comment:
I'm assuming that `-DLIBRETRO_STATIC=1` is not needed.

Reply:
I also assumed this and can at least confirm that it builds without and the libretro-core is capable of running games
Performance is not that good but that may depend on some missing features of panfrost driver